README.md

```diff
@@ -95,6 +95,14 @@ pnpm exec playwright install  # first time only
 pnpm exec playwright test
 ```
 
+To develop and debug end-to-end tests use
+
+```shell
+pnpm exec playwright test --ui --headed
```

Comment:
Perhaps add as "test" to package.json?

Reply:
done in 6f5dbbef843e5fa1531e26a661c08c688776e443